dotnet-testing/MockingWithNSubstitute/MockingWithNSubstitute.Tests/NotificationServiceTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using FluentAssertions;
+using NSubstitute;
+
+namespace MockingWithNSubstitute.Tests
+{
+    public class NotificationServiceTests
+    {
+        private readonly User _user;
+        private readonly IEmailService _emailService;
+        private readonly NotificationService _notificationService;
+
+        public NotificationServiceTests()
+        {
+            _user = new User("Code-Maze", "email@code-maze.com");
+            _emailService = Substitute.For<IEmailService>();
+            _notificationService = new NotificationService(_emailService);
+        }
+
+        [Fact]
+        public void GivenInputIsCorrect_WhenNotifyUserIsInvoked_ThenTrueIsReturned()
+        {
+            // Arrange            
```

Comment:
No need for the //Arrange //Act  //Assert comments

Reply:
I'd like to leave that just as a clarification if there are no other 0bjections.